docs/architecture/grpc-for-wcf-developers/create-project.md

```diff
@@ -93,28 +93,45 @@ In the gRPC build targets, there's a `Protobuf` item element that lets you speci
 The `GreeterService` class is in the `Services` folder and inherits from `Greeter.GreeterBase`. Rename it to `PortfolioService`, and change the base class to `Portfolios.PortfoliosBase`. Delete the `override` methods.
 
 ```csharp
-public class PortfolioService : Portfolios.PortfoliosBase
+public class PortfolioService : Protos.Portfolios.PortfoliosBase
 {
 }
 ```
 
-There was a reference to the `GreeterService` class in the `Configure` method in the `Startup` class. If you used refactoring to rename the class, this reference should have been updated automatically. However, if you didn't, you need to edit it manually.
+There was a reference to the `GreeterService` class in the `Program.cs`. If you used refactoring to rename the class, this reference should have been updated automatically. However, if you didn't, you need to edit it manually.
 
 ```csharp
-public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-{
-    if (env.IsDevelopment())
-    {
-        app.UseDeveloperExceptionPage();
-    }
+using TraderSys.Portfolios.Services;
 
-    app.UseRouting();
+var builder = WebApplication.CreateBuilder(args);
 
-    app.UseEndpoints(endpoints =>
-    {
-        endpoints.MapGrpcService<PortfolioService>();
-    });
-}
+// Add services to the container.
+
+builder.Services.AddGrpc();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+app.MapGrpcService<PortfolioService>();
+app.MapGet("/", () => "Communication with gRPC endpoints must be made through a gRPC client. To learn how to create a client, visit: https://go.microsoft.com/fwlink/?linkid=2086909");
+
+app.Run();
+```
+
+Using a `GlobalUsing` file, which applies to all the source file in the project and includes all the namespaces makes it easy to maintain them at one place
```

Comment:
For this sample, we don't require a `GlobalUsing` file. Can you drop this one, please?

Reply:
Yes sure, I have dropped GlobalUsing